src/__tests__/tv.test.ts

```diff
@@ -2093,6 +2093,48 @@ describe("Tailwind Variants (TV) - Extends", () => {
     expect(result).toHaveClass(expectedResult);
   });
 
+  test("should override the extended classes with variants, both using array", () => {
+    const p = tv({
+      base: "text-base text-green-500",
+      variants: {
+        isBig: {
+          true: "text-5xl",
+          false: "text-2xl",
+        },
+        color: {
+          red: ["text-red-500 bg-red-100", "tracking-normal"],
+          blue: "text-blue-500",
```

Comment:
I think this test should not be just this case, but rather we should test all the combinations, e.g.

- extend: array, other: array
- extend: array, other: string
- extend: string, other: array
- extend: string, other: string

We can do this in the same test just using different variant values.